tasks/st0317a_clean_cert.rb

```diff
@@ -45,7 +49,11 @@ def clean_cert(agent, cmd)
   }
 end
 
-results = {}
+deprecation_msg = "This task is deprecated and has been replaced by the certificate clean api, which provides the same functionality. 
```

Comment:
There's still a warning about whitespace on this line.  A `pdk validate -a` should fix it.